submissions/abhishekkukreja/about.md

```diff
@@ -0,0 +1,11 @@
+# About Me
+My name is Abhishek. I am a CSE student at Panjab University, India. 
+# Experience
+Not yet. 
```

Comment:
Add more experiences, this is just to let you practice squashing of commits. @abhishekkukreja